qermit/probabilistic_error_cancellation/pec_learning_based.py

```diff
@@ -712,15 +712,16 @@ def label_gates(circ: Circuit) -> Circuit:
     frame_count = 0
     for command in command_list:
         labelled_command = command.copy()
-        if labelled_command["op"]["type"] in ("U1", "U2", "U3"):
+        print("gate type ====== ", labelled_command["op"]["type"])
```

Comment:
This seems like a left over from my tests. It should probably go. Sorry about that!